actions.json

```diff
@@ -0,0 +1,20 @@
+[{
+        "id": "update_turnserver_ip",
+        "name": "Update turnserver ip",
+        "command": "/opt/yunohost/matrix-$YNH_APP_ID/Coturn_config_rotate.sh",
+        "user": "root",
+        "accepted_return_codes": [0],
+        "description": {
+            "en": "Update the ip in the turnserver config"
+        }
+},
+{
+        "id": "open_turnserver_firewall_ports",
+        "name": "Open ports for turnserver",
+        "command": "yunohost firewall allow Both 49153:49193",
```

Comment:
What about another one to close those ports ?

Reply:
Yes, I could do it if you want